_rules/SC2-4-9-links-with-identical-accessible-names-have-same-purpose.md

```diff
@@ -0,0 +1,234 @@
+---
+name: Links with identical accessible names have same purpose
+test_type: atomic
+
+description: |
+  This rule checks that identical accessible names are only used for links that have the same purpose
+
+success_criterion: 
+- 2.4.9 # Link Purpose (Link Only)
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Anne Thyme Nørregaard
+---
+
+## Test Procedure
+
+### Applicability
+
+This rule applies to any two or more HTML or SVG elements that have the [semantic role](#semantic-role) of link, are [exposed to assistive technologies](#exposed-to-assistive-technologies) and that have the same [accessible name](#accessible-name).
+
+### Expectation
+
+When activated, the links in each set of target elements resolve to resources that fulfill the same purpose indicated by the [accessible names](#accessible-name) of the link.
+
+**Note:** Resolving the links includes potential redirects.
+
+**Note:** Web pages and documents (e.g. PDFs, office formats etc.) may fulfil the same purpose in relation to the link, even if the resources:
+* are located on different URLs, including different domains
+* present different navigation options, e.g. through bread crumbs or local sub menus
+* contain different amounts of information and/or differently worded information
+* are using different layouts.
+
+**Note:** If the same content is presented in different formats, the format itself is often part of the link purpose, e.g. an article as both HTML and PDF.
+
+**Note:** If the [normalized](#url-normalisation) value of the ´href´ is identical, the resources are identical, thus fulfilling the same purpose. [Relative URLs](https://www.w3.org/TR/WD-html40-970917/htmlweb.html#relative-urls) first need to be resolved to full URLs before doing the comparison.
+
+## Assumptions
+
+* This rule assumes that the purpose of the links for links with identical link texts would not be ambiguous to users in general.
+
+## Accessibility support
+
+There are no major accessibility support issues known for this rule.
+
+## Background
+
+- [Understanding Success Criterion 2.4.9: Link Purpose (Link Only)](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-link-only.html)
+- [URL normalization](#url-normalisation)
+
+## Test Cases
+
+### Passed
```

Comment:
Several test cases could be used exactly as they are written but as examples of failed test cases because they are dependent on the content of the files. Don't we need to provide the linked files also for the test cases to be meaningful?

Reply:
@carlosapaduarte, yes, this is part of the work load left for me on this one.